docs/Running-Tachyon-Fault-Tolerant.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: global
+title: Tachyon Standalone with Fault Tolerance
+nickname: Tachyon Standalone with Fault Tolerance
+group: User Guide
+priority: 3
+---
+
+Fault Tolerance in Tachyon is based upon a multi-master approach where multiple master processes
+are run. One of these processes is elected the leader and is used by all workers and clients as the
+primary point of contact. The other masters act as standbys using the shared journal to ensure that
```

Comment:
Would be better to specify that `the shared journal` is generated by leader master, other masters just read the journal to keep the same state with master, also, it's good to know where the journal should be put.